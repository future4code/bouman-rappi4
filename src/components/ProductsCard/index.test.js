import React from 'react'
import renderer from "react-test-renderer";
import { ProductCard } from './index';

describe ('teste componente ProductCard', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<ProductCard></ProductCard>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})