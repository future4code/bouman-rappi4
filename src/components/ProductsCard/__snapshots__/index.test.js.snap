// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente ProductCard renders correctly 1`] = `
<div
  className="sc-bdVaJa klyngg"
>
  <img
    alt="image"
    className="sc-bwzfXH dKXYyA"
  />
  <div
    className="sc-htpNat jdcmRH"
  >
    <h3
      className="sc-bxivhb bOgZRB"
    />
    <div
      className="sc-ifAKCX ftMmuE"
    />
    <div
      className="sc-EHOje gzyEmj"
    >
      R$ 
    </div>
  </div>
  <button
    onClick={[Function]}
  >
     Adicionar
  </button>
</div>
`;
